modules/packages/Image.chpl

```diff
@@ -39,6 +39,19 @@ and written to a second BMP file.
    writeImage("pixels.bmp", imageType.bmp, arr);
    writeImage("pixels2.bmp", imageType.bmp, scale(arr, 2));
 
+In another example, the following code reads a PNG file, removes all green from
+the image, and writes it back out to a new JPG file.
+
+.. code-block:: chapel
+
+   use Image;
+
+   var arr = readImage("input.png", imageType.png);
+   const fmt = (rgbColor.red, rgbColor.green, rgbColor.blue);
+   var colors = pixelToColor(arr, format=fmt);
+   [x in colors] c(1) = 0;
+   arr = colorToPixel(colors, format=fmt);
+   writeImage("output.jpg", imageType.jpg, arr);
```

Comment:
I suspect you may want an example like this in `test` to make sure it works, with a backlreference to the docs.

Reply:
Good call, since there was a typo in the example 😄